docs/_documentations/codewind-eclipse-quick-guide.md

```diff
@@ -0,0 +1,167 @@
+
+Eclipse Codewind provides the ability to create projects based on a variety of different template types.  These include IBM Cloud starters, OpenShift Do (odo), and Appsody templates. Today, there are templates for: IBM Cloud Starters, odo, Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, Node.js with Loopback.
+
+## Developing with Eclipse 
+If you use Eclipse, you can use Codewind for Eclipse to develop and debug your containerized projects from within a local Eclipse IDE.
+
+### Prerequisite
+Before you can develop a microservice with Eclipse, you need to:
+
+* [Install Docker](https://docs.docker.com/install/) 
+    * **Note:** Make sure to install or upgrade to minimum Docker version 19.03. 
+* [Install Eclipse](https://www.eclipse.org/downloads/packages/release/)
+    * **Note:** Make sure to install or upgrade to mimimum Eclipse version 2019-09 R (4.13.0). 
+
+### Installing Codewind for Eclipse
+The Codewind installation pulls the following images that form the Codewind backend:
+
+1. `eclipse/codewind-performance-amd64`
+2. `eclipse/codewind-pfe-amd64`
+
+The Codewind installation includes two parts:
+
+1. The Eclipse plug-in installs when you install Codewind from the [Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind)or when you install by searching in the `Eclipse Extensions` view.
+2. The Codewind backend containers install after you click `Install`. Clicking `Install` downloads the Codewind backend containers, ~1GB. 
+
+### Configuring Codewind to use application stacks
+Configure Codewind to use Appsody templates so you can focus exclusively on your code. These templates include an Eclipse MicroProfile stack that you can use to follow this guide. Complete the following steps to select the Appsody templates:
+
+1. Click the `Codewind` tab. 
+2. Expand `Codewind` by clicking the drop-down arrow.
+3. Right-click `Local [Running]`.
+4. Select `Manage Template Sources...`. 
+5. Select `Appsody Stacks - incubator`.
+6. Click the `OK` button. 
+
+You now have configured Codewind to Appsody templates and can proceed to develop your microservice within Codewind.
+
+If your organization uses customized application stacks and has given you a URL that points to an `index.json` file, you can add it to Codewind: 
+
+1. Return to  `Codewind` and right-click `Local [Running]`. 
+2. Select `Manage Template Sources...`. 
+3. Click the `Add...` button to add your URL.
+4. Add your URL in the `URL:` box in the pop up window and save your changes. 
+
+### Creating an Appsody project
+Appsody helps you develop containerized applications and removes the burden of managing the full software development stack. If you want more context about Appsody, visit the https://appsody.dev/docs[Appsody welcome page]. 
```

Comment:
Correct link syntax at end of this sentence to `[Appsody welcome page](https://appsody.dev/docs)`